x-pack/libbeat/processors/add_cloudfoundry_metadata/docs/add_cloudfoundry_metadata.asciidoc

```diff
@@ -0,0 +1,51 @@
+[[add-cloudfoundry-metadata]]
+[role="xpack"]
+=== Add Cloud Foundry metadata
+
+experimental[]
+
+The `add_cloudfoundry_metadata` processor annotates each event with relevant metadata
+from Cloud Foundry applications. The events are annotated with Cloud Foundry metadata,
+only if the event contains a reference to a Cloud Foundry application and the configured
+Cloud Foundry client is able to retrieve information for the application.
```

Comment:
It would be nice to explicitly explain what field (`cf.app.id`) is used for correlation. So people wanting to do their own enrichment have everything they need.

Reply:
Okay will add, it will become `cloudfoundry.app.id`. Should a field be added to the config to adjust this field path? Don't think its a requirement, but easy to add, might be have a use case?

Reply:
We have done this in the past, ie: https://github.com/elastic/beats/blob/3a764e634cf11642cc718f14844f487669140120/libbeat/processors/add_process_metadata/config.go#L39. That said, I'm fine with having a more opinionated behavior here. Nowadays people can use the rename processor to adjust the doc before enriching (this wasn't possible in the past).